docs/markdown/bar-series.md

```diff
@@ -117,3 +117,7 @@ The opacity for all elements in the series, this property will be over-ridden by
 #### stroke
 Type: `string|number`
 The outer color for all elements in the series, this property will be over-ridden by color specified in the data attribute.
+
+### style
+Type: `object`
+A list of CSS properties to style the series outside of the explicitly set properties. Note that it will override all other properties (ie fill, stroke, opacity, color).
```

Comment:
1) we can iterate on the wording. React doc (which is about the same concept) doesn't use either. They specify, however, that the properties are camel cased and not hyphenated. I'm not saying that we should use the react docs wording, but rather come up with something which is completely clear and unambiguous. 
2) I disagree with this though. Possible API attributes should be repeated, and possibly rephrased, for each and every component. The style attribute should also has its own page that goes into more detail. But eventually, for every component, such as every series type, there should be an exhaustive list of all the attributes that can be passed, phrased in the way that makes the most sense in this context. When a user wants to know how to use, ie, LineSeries, they should have access to the entirety of information on how to use it, as opposed to have to jump back and forth between pages.